context.go

```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"net/http"
 
+	"go.opentelemetry.io/otel/attribute"
```

Comment:
I'm not too keen on adding OTel as a dependency for every user. Would it be reasonable to use `map[string]interface{}` in this code, and convert from `go.opentelemetry.io/otel/attribute` in apmotel?